mockingServer/modules/LoggingSession.py

```diff
@@ -0,0 +1,17 @@
+from mockingServer.modules.RequestHandler import RequestHandler
+from flask import Blueprint, make_response
+import json
+
+loggingBluePrint = Blueprint('LoggingService', import_name=__name__)
+
+
+class LoggingService(RequestHandler):
+    def post(self):
+        if not self.check_session_id():
+            return make_response('sessionID check failed', 401)
+        with open('mockingServer/data/logging.json') as data:
```

Comment:
either use `r` everywhere for opening read only or skip it, you have mixed it.

Reply:
Will add 'r' for better readability.

Reply:
done